html/services_section.html

```diff
@@ -0,0 +1,81 @@
+<section id='services_section'>
+  <span id='services_anchor' class='anchors'></span>
+  <h2>Services</h2>
+  <!-- <div class='services_sections' id='services_sec_workshops'> -->

```

Comment:
you should change your html structure for each of the services cells.
then you wouldn't have to use so many classes.

```suggestion
  <!-- <div class='services_sections' id='services_sec_workshops'> -->
<div>
	<h2></h2>
	<img>
	<h3></h3>
	<p></p>
	<h3></h3>
	<p></p>
	<a></a>
</div>
```